internal/logger/logrotate.go

```diff
@@ -18,14 +18,14 @@
 package logger
 
 import (
-	"compress/gzip"
 	"encoding/json"
 	"fmt"
 	"io"
 	"os"
 	"path/filepath"
 	"time"
 
+	"github.com/klauspost/compress/gzip"
```

Comment:
I know that guy.

Reply:
I know him. He is too lazy to add a zstd option, though ;)